prod/Dockerfile

```diff
@@ -10,11 +10,14 @@ COPY ../libs/db/src/prisma/ ./prisma/
 
 RUN chown -R backend:backend .
 
+USER backend
 RUN npm --omit=dev --ignore-scripts install
 RUN node_modules/.bin/prisma generate --schema ./prisma/schema.prisma
 
 RUN rm package.json package-lock.json
 COPY ./prod/package.json .
 
+RUN chown -R backend:backend ./package.json
+
```

Comment:
COPY actually has a `--chown` flag: https://docs.docker.com/reference/dockerfile/#copy---chown---chmod so this isn't needed if you do that